torch/onnx/symbolic_opset9.py

```diff
@@ -1903,21 +1903,28 @@ def new_full(g, self, size, fill_value, dtype, layout, device, pin_memory=False)
 
 
 def eye(g, *args):
```

Comment:
Since the number of args is fixed now, can you please change this from `*args` to `n, m, k, dtype, layout, device, pin_memory`?

Reply:
not done?

Reply:
Sorry, pushed a bad commit. Fixed.